Loop/Managers/LoopDataManager.swift

```diff
@@ -310,6 +294,7 @@ class LoopDataManager {
             predictedGlucose = nil
         }
     }
+    private var insulinOnBoard: InsulinValue?
```

Comment:
I think it's reasonable to have (momentary) IOB available here, because it can be helpful for implementing alternative dosing approaches.